aws_marketplace/using_algorithms/automl/AutoML_-_Train_multiple_models_in_parallel.ipynb

```diff
@@ -0,0 +1,635 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Goal: Train a model using Automl functionality! \n",
```

Comment:
s/Automl/AutoML

Reply:
Please make sure the above casing is used everywhere.

Reply:
Sure, thanks for catching it.